docs/changes/utils/14624.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Ensure masks are propagated correctly for ``outer`` methods of ufuncs also if
+one of the inputs is not actually masked.
```

Comment:
Since #14590 hasn't been released yet is this message still relevant?

Reply:
Yes, this was a bug that already existed; I noticed it when I was thinking about the masks for #14590.